engines/sherlock/metaengine.cpp

```diff
@@ -252,6 +289,958 @@ SaveStateDescriptor SherlockMetaEngine::querySaveMetaInfos(const char *target, i
+		act->setCustomEngineActionEvent(kActionTattooPasswordRight);
+		act->addDefaultInputMapping("RIGHT");
+		act->addDefaultInputMapping("KP6");
+		act->addDefaultInputMapping("JOY_RIGHT");
+		tattooPasswordKeymap->addAction(act);
+
+		// I18N: (Game name: The Lost Files of Sherlock Holmes: The Case of the Rose Tattoo) This action is used to go to the start of the password input field
+		act = new Action("GOSTART", _("Go to start of password"));
+		act->setCustomEngineActionEvent(kActionTattooPasswordStart);
+		act->addDefaultInputMapping("HOME");
+		act->addDefaultInputMapping("KP7");
+		tattooPasswordKeymap->addAction(act);
+
+		// I18N: (Game name: The Lost Files of Sherlock Holmes: The Case of the Rose Tattoo) This action is used to go to the end of the password input field
+		act = new Action("GOEND", _("Go to end of password"));
+		act->setCustomEngineActionEvent(kActionTattooPasswordEnd);
+		act->addDefaultInputMapping("END");
+		act->addDefaultInputMapping("KP1");
+		tattooPasswordKeymap->addAction(act);
+
+		// I18N: (Game name: The Lost Files of Sherlock Holmes: The Case of the Rose Tattoo) save password input field has 2 modes, insert and overwrite, this action toggles between them
+		act = new Action("INSERTMODE", _("Toggle insert mode"));
+		act->setCustomEngineActionEvent(kActionTattooPasswordToggleInsertMode);
+		act->addDefaultInputMapping("INSERT");
+		act->addDefaultInputMapping("JOY_UP");
+		tattooPasswordKeymap->addAction(act);
+
+		act = new Action("NEXTOPTION", _("Select next option"));
+		act->setCustomEngineActionEvent(kActionTattooQuitDialogNextOption);
+		act->addDefaultInputMapping("TAB");
+		tattooQuitDialogKeymap->addAction(act);
+
+		act = new Action("YES", _("Yes"));
+		act->setCustomEngineActionEvent(kActionTattooQuitDialogYes);
+		act->addDefaultInputMapping("y");
+		act->addDefaultInputMapping("JOY_X");
+		tattooQuitDialogKeymap->addAction(act);
+
+		act = new Action("NO", _("No"));
+		act->setCustomEngineActionEvent(kActionTattooQuitDialogNo);
+		act->addDefaultInputMapping("n");
+		act->addDefaultInputMapping("ESCAPE");
+		act->addDefaultInputMapping("JOY_Y");
+		tattooQuitDialogKeymap->addAction(act);
+
+		act = new Action("NEXT", _("Go to next talk option"));
+		act->setCustomEngineActionEvent(kActionTattooTalkNext);
+		act->addDefaultInputMapping("TAB");
+		act->addDefaultInputMapping("JOY_DOWN");
+		tattooTalkKeymap->addAction(act);
+
+		act = new Action("PREVIOUS", _("Go to previous talk option"));
+		act->setCustomEngineActionEvent(kActionTattooTalkPrevious);
+		act->addDefaultInputMapping("S+TAB");
+		act->addDefaultInputMapping("JOY_UP");
+		tattooTalkKeymap->addAction(act);
+
+		act = new Action("EXIT", _("Exit talk window"));
+		act->setCustomEngineActionEvent(kActionTattooTalkExit);
+		act->addDefaultInputMapping("ESCAPE");
+		act->addDefaultInputMapping("JOY_Y");
+		tattooTalkKeymap->addAction(act);
+
+		act = new Action("EXIT", _("Exit verb menu"));
```

Comment:
Shouldn't it be generit "Exit" or "Close window"? So then we reduce number of translateable strings